bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/camera.xml

```diff
@@ -77,40 +88,58 @@
 		<label>Is Local</label>
 		<description>Only for scope access_camera. If Camera and application requesting the information are on the same
 			network (true/false)</description>
+		<tags>
+			<tag>Status</tag>
+		</tags>
 		<state readOnly="true"></state>
 	</channel-type>
 
 	<channel-type id="live_picture_url" advanced="true">
 		<item-type>String</item-type>
 		<label>Live Snapshot URL</label>
 		<description>Url of the live snapshot for this camera (need scope access_camera)</description>
+		<tags>
+			<tag>Status</tag>
+		</tags>
 		<state readOnly="true"></state>
 	</channel-type>
 
 	<channel-type id="live_picture">
 		<item-type>Image</item-type>
 		<label>Live Snapshot</label>
 		<description>Camera Live Snapshot</description>
+		<tags>
+			<tag>Status</tag>
+		</tags>
 		<state readOnly="true"></state>
 	</channel-type>
 
 	<channel-type id="live_stream_url" advanced="true">
 		<item-type>String</item-type>
 		<label>Live Stream URL</label>
 		<description>Url of the live stream for this camera</description>
+		<tags>
+			<tag>Status</tag>
+		</tags>
 		<state readOnly="true"></state>
 	</channel-type>
 
 	<channel-type id="floodlightAutoMode">
 		<item-type>Switch</item-type>
 		<label>Floodlight Auto-Mode</label>
 		<description>State of the floodlight auto-mode</description>
+		<tags>
+			<tag>Switch</tag>
+		</tags>
 	</channel-type>
 
 	<channel-type id="floodlight">
 		<item-type>Switch</item-type>
 		<label>Floodlight</label>
 		<description>State of the floodlight</description>
+		<tags>
+			<tag>Switch</tag>
```

Comment:
Can you help me understand what the "Switch" tag was good for? This channel seems to be related to light, so I'd rather would have expected a tag for that. Just because something is "switchable" is imho not enough for such a tag, especially as this information is already contained in the item-type.

Reply:
To tell the truth @kaikreuzer , I did not spend much time reviewing this because existing version of the Netatmo binding is hardly functional due to some evolutions on Netatmo API side. I rewrote it completely (opened PR on this) still WIP.

Reply:
Yes, I agree, "Light" should be added.

Reply:
But then again, "Light" is usually used to light up some room. A floodlight of a camera has a fairly special semantic and shouldn't be used as a random light (you don't want it to go on when you enter the room. You probably do not want it to turn off, when you're leaving the house, etc.)
I'd in such cases hence rather prefer to not put any tag on it, but leave it to the user on how he actually uses the device. Only he knows his concrete use cases.

Reply:
Note that there a few other "Switch" tags in that PR as well.

Reply:
I just look at Netatmo product information, the outside camera integrates a real light, it combines a light and a camera.
https://www.netatmo.com/fr-fr/security/cam-outdoor
Considering it as a light that can be switch on and off is IMHO fully correct as it is what it is.

Reply:
I checked again all channels with only "Switch" tag and fixed them.